packages/odyssey/src/scss/abstracts/_colors.scss

```diff
@@ -55,6 +55,7 @@ $colors: (
     '500': #e97107
   ),
   'turquoise': (
+    '400': #65adb7,
```

Comment:
Has this been added to the Color docs? They're getting an update, but we should have a content placeholder for the change if we don't.

Reply:
Good catch, it has not. Adding now.